test/ch/epfl/leb/sass/simulator/loggers/FrameLoggerTest.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2017 Laboratory of Experimental Biophysics
+ * Ecole Polytechnique Federale de Lausanne
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package ch.epfl.leb.sass.simulator.loggers;
+
+import ch.epfl.leb.sass.simulator.loggers.StateLogger;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Rule;
+import org.junit.rules.TemporaryFolder;
+import static org.junit.Assert.*;
+
+/**
+ * Logs all state transitions from a simulation.
+ * @author Kyle M. Douglass
+ */
+public class StateLoggerTest { 
```

Comment:
This should be renamed to FrameLoggerTest since Travis identifies this as a duplicate of the real StateLoggerTest class.